flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/calcite/RelTimeIndicatorConverter.java

```diff
@@ -544,11 +595,110 @@ private RelNode convertAggInput(Aggregate agg) {
                 .collect(Collectors.toList());
     }
 
+    private FlinkLogicalWindowAggregate visitWindowAggregate(FlinkLogicalWindowAggregate agg) {
```

Comment:
are there any cases which can cover the change?

Reply:
MatchRecognizeTest#testMatchRecognizeOnRowtimeLTZ

Reply:
It's better we can add some tests in WindowAggregateTest to cover these changes

Reply:
The case cover the branch must contain a Match with Match_ROWTIME, the successor is a WindowAggregate, which is exactly what `MatchRecognizeTest#testMatchRecognizeOnRowtimeLTZ` do. I think we don't need to add extra tests.

Reply:
make sense